src/models/User.ts

```diff
@@ -203,9 +209,17 @@ export const userRouter = router({
     }),
   setEmailValidated: publicProcedure
     .input(z.boolean())
-    .query(async ({ input, ctx }) => {
+    .mutation(async ({ input, ctx }) => {
       await ctx.state.storage.put(StorageKeys.emailValidated, input);
     }),
+  setPassword: publicProcedure
+    .input(z.string())
```

Comment:
do we have any validation on this? personally I don't like our current auth0 password validation as per XKCD's `correcthorsebatterystaple`

Reply:
You mean like password strength? Currently now but should be added for sure

Reply:
yes that's exactly what i meant :+1: